internal/xcuitest/config.go

```diff
@@ -42,9 +42,12 @@ type Project struct {
 
 // Xcuitest represents xcuitest apps configuration.
 type Xcuitest struct {
-	App       string   `yaml:"app,omitempty" json:"app"`
-	TestApp   string   `yaml:"testApp,omitempty" json:"testApp"`
-	OtherApps []string `yaml:"otherApps,omitempty" json:"otherApps"`
+	App                  string   `yaml:"app,omitempty" json:"app"`
+	AppDescription       string   `yaml:"appDescription,omitempty" json:"appDescription"`
+	TestApp              string   `yaml:"testApp,omitempty" json:"testApp"`
+	TestAppDescription   string   `yaml:"testAppDescription,omitempty" json:"testAppDescription"`
+	OtherApps            []string `yaml:"otherApps,omitempty" json:"otherApps"`
+	OtherAppDescriptions []string `yaml:"otherAppDescriptions,omitempty" json:"otherAppDescriptions"`
```

Comment:
Shouldn't it be called this way?

```suggestion
	OtherAppsDescriptions []string `yaml:"otherAppsDescriptions,omitempty" json:"otherAppsDescriptions"`
```